user/docker.md

```diff
@@ -137,6 +137,20 @@ Within your `.travis.yml` prior to attempting a `docker push` or perhaps before
 docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 ```
 
+#### Branch Based Registry Pushes
+
+Using `after_success` it is also possible to push an image to a registry based
+on the branch:
+
```

Comment:
I'd rephrase this to something like:

    To push a particular branch of your repository to a remote registry, use the `after_success` section of your `.travis.yml`:

> you also don't need to indent fenced code blocks.